docs/Quick-Start.md

```diff
@@ -20,7 +20,7 @@ VictoriaMetrics is distributed in two forms:
 
 Single-server-VictoriaMetrics VictoriaMetrics is available as:
 
-* [Managed VictoriaMetrics at AWS](https://aws.amazon.com/marketplace/pp/prodview-4tbfq5icmbmyc)
+* [VictoriaMetrics Cloud at AWS](https://aws.amazon.com/marketplace/pp/prodview-4tbfq5icmbmyc)
```

Comment:
`[VictoriaMetrics Cloud](https://cloud.victoriametrics.com/signUp?utm_source=website&utm_campaign=docs_vm_quickstart) – the most cost-efficient hosted monitoring platfor, operated by VictoriaMetrics core team.`

Reply:
fixed